app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -0,0 +1,70 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
+    require_nested :RefreshParser
+    require_nested :RefreshWorker
+    require_nested :Refresher
+
+    include AuthenticationMixin
+
+    DEFAULT_PORT = 80
+    default_value_for :port, DEFAULT_PORT
+    has_many :middleware_servers, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :middleware_deployments, :foreign_key => :ems_id, :dependent => :destroy
+
+    def verify_credentials(_auth_type = nil, _options = {})
```

Comment:
It is customary to only prefix variables with underscore that are NOT used.  So, in this case, `_options` should become `options` since it is used in this method, whereas `_auth_type` can remain as is because it is not used.